src/main/java/com/segment/analytics/android/integrations/branch/BranchIntegration.java

```diff
@@ -1,40 +1,66 @@
 package com.segment.analytics.android.integrations.branch;
 
 import android.content.Context;
+import android.text.TextUtils;
 
 import com.segment.analytics.Analytics;
 import com.segment.analytics.ValueMap;
+import com.segment.analytics.integrations.AliasPayload;
 import com.segment.analytics.integrations.IdentifyPayload;
 import com.segment.analytics.integrations.Integration;
 import com.segment.analytics.integrations.Logger;
 import com.segment.analytics.integrations.ScreenPayload;
 import com.segment.analytics.integrations.TrackPayload;
 
+import org.json.JSONArray;
+import org.json.JSONException;
 import org.json.JSONObject;
 
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Iterator;
+
+import io.branch.indexing.BranchUniversalObject;
 import io.branch.referral.Branch;
+import io.branch.referral.Defines;
+import io.branch.referral.util.BRANCH_STANDARD_EVENT;
+import io.branch.referral.util.BranchEvent;
+import io.branch.referral.util.CurrencyType;
 
 /**
  * @author sahilverma
- * @date 2015-11-05
+ * 2015-11-05
```

Comment:
Why did you remove the @date tag?

Reply:
This was causing a java doc warning since @date is not a standard java doc tag